profiles/car.lua

```diff
@@ -206,10 +206,10 @@ local function getTurnLanes(way)
     local turn_lanes_bw = way:get_value_by_key("turn:lanes:backward")
 
     if( fw_psv ~= 0 or bw_psv ~= 0 ) then
-        turn_lanes = trimLaneString(turn_lanes, bw_psv, fw_psv )
-        turn_lanes_fw = trimLaneString(turn_lanes_fw, bw_psv, fw_psv )
+        if( turn_lanes ) then turn_lanes = trimLaneString(turn_lanes, bw_psv, fw_psv ) end
+        if( turn_lanes_fw ) then turn_lanes_fw = trimLaneString(turn_lanes_fw, bw_psv, fw_psv ) end
         --backwards turn lanes need to treat bw_psv as fw_psv and vice versa
-        turn_lanes_bw = trimLaneString(turn_lanes_bw, fw_psv, bw_psv )
+        if( turn_lanes_bw) then turn_lanes_bw = trimLaneString(turn_lanes_bw, fw_psv, bw_psv ) end
```

Comment:
+        if( turn_lanes_bw) then turn_lanes_bw = trimLaneString(turn_lanes_bw, fw_psv, bw_psv ) end.... not work.

i change 208 string: if( fw_psv and fw_psv ~= 0 or bw_psw and bw_psv ~= 0 ) then